src/vaadin-menu-bar-interactions-mixin.html

```diff
@@ -75,16 +75,6 @@
       document.removeEventListener('click', this.__boundOutsideClickListener, true);
     }
 
-    /**
-     * Can be called to manually notify a resizable and its descendant
-     * resizables of a resize change.
-     */
-    notifyResize() {
-      // NOTE: we have this method here to include it to TypeScript definitions.
-      // gen-typescript-declarations does not generate types for `mixinBehaviors`
-      super.notifyResize();
-    }
-
```

Comment:
According to the comment, looks like this is still needed in the P2 version

Reply:
Done.